cmd/provenanced/cmd/root.go

```diff
@@ -242,6 +238,11 @@ func txCommand() *cobra.Command {
 func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts servertypes.AppOptions) servertypes.Application {
 	var cache sdk.MultiStorePersistentCache
 
+	timeoutCommit := cast.ToDuration(appOpts.Get("consensus.timeout_commit"))
+	if timeoutCommit < 4*time.Second {
```

Comment:
Should this be `5*time.Second`? It looks like 4 seconds can get around the error.

Reply:
Our main goal is to alert folks that are using the (old) default of `1s`. We know a value of `1s` isn't good. And we're pretty sure a value of `5s` is the way to go. But we don't really know about the values in-between. I originally chose `4s` here so that the error didn't show up for something they've chosen that is less than `5s`, but probably still okay, e.g. `4300ms`.

I've got a change coming, though, that only shows this error if it's less than half the default value, and only if running on mainnet.